usecases/payload_parser/payload_test.go

```diff
@@ -103,20 +125,82 @@ func TestParser_ValidatePayload(t *testing.T) {
 			input: []byte(`{"unknown": "unknown"}`),
 			err:   errIsInvalidDataType,
 		},
+		{
+			name: "nullable fields without object_id and updated_at",
+			table: models.Table{
+				Name: "transactions",
+				Fields: map[models.FieldName]models.Field{
+					"nullable": {
+						DataType: models.String,
+						Nullable: true,
+					},
+				},
+			},
+			input: []byte(`{}`),
+			want: models.ClientObject{
+				TableName: "transactions",
+				Data: map[string]any{
+					"nullable": nil,
+				},
+			},
+			wantErrors: map[models.FieldName]string{
+				"object_id":  errIsNotNullable.Error(),
+				"updated_at": errIsNotNullable.Error(),
+			},
+		},
+		{
+			name: "nullable fields with object_id and updated_at",
+			table: models.Table{
+				Name: "transactions",
+				Fields: map[models.FieldName]models.Field{
+					"nullable": {
+						DataType: models.String,
+						Nullable: true,
+					},
+					"object_id": {
+						DataType: models.String,
+						Nullable: false,
+					},
+					"updated_at": {
+						DataType: models.Timestamp,
+						Nullable: false,
+					},
+				},
+			},
+			input: []byte(`{"object_id": "id", "updated_at": "2023-10-19T00:00:00+03:00"}`),
+			want: models.ClientObject{
+				TableName: "transactions",
+				Data: map[string]any{
+					"nullable":  nil,
+					"object_id": "id",
+					// input is in UTC+3, but the output is in UTC
+					"updated_at": time.Date(2023, time.October, 18, 21, 0, 0, 0, time.UTC),
```

Comment:
note this test case

Reply:
What about a non TZ timestamp (the new format) ? How does it behaves when the server do `t.UTC()` does it return a TZ based on the server one ? or does it infer it's UTC by default ?

Reply:
Yes it parses to UTC